library/swift_disk.py

```diff
@@ -4,22 +4,31 @@
 DOCUMENTATION = '''
 ---
 author: Jesse Keating and Dustin Lundquist
-module: cinder_volume_group
-short_description: Configure LVM volume groups
+module: swift_disk
+short_description: Configure Swift disks/partition
 description:
-  - This module creates a LVM VG for Cinder to use
+  - This module prepares the disk/partition for use with Swift
 options:
   dev:
     description:
       - The device to make a swift disk
-    required: true
+    required: false
+  partition_path:
+    description: 
+      - Complete path to partition to be used for Swift
+    required: false
 '''
 
 EXAMPLES = '''
-# create a volume group for cinder, named `cinder-volumes`,
-# backed by a file mounted over loop device
+# use the sdb disk for swift. it will be formatted to xfs
+# and mounted on /srv/node/sdb
 
 - swift_disk: dev=sdb
+
+# use the lvm disk /dev/vgpool/sftmeta. Partition will be formatted
+# and mounted on /srv/node/sftmeta
+
+- swift_disk: partition_path=/srv/node/sftmeta
```

Comment:
I don't understand this example. partition_path says a complete path to the partition, but the value you give here is a path to where it's to be mounted.